test/TensorFlow/retain_release.swift

```diff
@@ -121,13 +121,7 @@ public func testBalancedRetainReleases() {
 // CHECK: function_ref @_swift_tfc_FinishTensorComputation
 // CHECK: [[H:%.*]] = alloc_ref $TensorHandle<Float>
 //
-// TFPartition pass generates these 2 strong_retain's to balance retain/releases.
```

Comment:
Same.

Reply:
Yes this should be fine as it's add and removing the same # retain/release insts.